tutorial/patterns.md

```diff
@@ -91,3 +91,11 @@ the model cannot be provided in the constructor is the initialization
 required for a model may have to call methods that are subclassed.
 The subclassed methods would be called before the subclass constructor has
 finished evaluating, resulting in undefined state.
+
+
+## Data Structures
+
+Prefer to use Phosphor Vectors over JavaScript Arrays for internal use,
+and expose Vectors as ISequences for external use.  This allows us to use
+the Phosphor algorithms which provide advanced functionality over what is
+offered by the builtin Arrays.
```

Comment:
I'd say "native" or "built-in". Sorry for being nit picky re: verbiage. Feel free to disregard.